Source/SIMPLib/CoreFilters/MoveMultiData.cpp

```diff
@@ -195,19 +194,17 @@ void MoveMultiData::dataCheck()
       // Number of tuples does not match
       if(daDestAttributeMatrix->getNumberOfTuples() != daSrcDataArray->getNumberOfTuples())
       {
-        setErrorCondition(-11019);
         QString ss = QObject::tr("The number of tuples of source Attribute Array (%1) and destination Attribute Matrix (%2) do not match")
-          .arg(daSrcDataArray->getNumberOfTuples())
-          .arg(daDestAttributeMatrix->getNumberOfTuples());
-        notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
+                         .arg(daSrcDataArray->getNumberOfTuples())
+                         .arg(daDestAttributeMatrix->getNumberOfTuples());
+        notifyErrorMessage("", ss, -11019);
         return;
       }
       // Source and Destination match
       if(daSrcPaths[i].hasSameAttributeMatrix(amDestPath))
       {
-        setWarningCondition(-11020);
-        QString ss = QObject::tr("The source and destination Attribute Matrix are the same.  Is this what you meant to do?");
-        notifyWarningMessage(getHumanLabel(), ss, getWarningCondition());
+                QString ss = QObject::tr("The source and destination Attribute Matrix are the same.  Is this what you meant to do?");
+        notifyWarningMessage("", ss, -11020);
```

Comment:
Check MoveData.cpp and MoveMultiData.cpp.  I think you are using tabs, or your formatting is just messed up.

Reply:
Yeah I did a grep for every place that uses setErrorCondition/notifyErrorMessage and setWarningCondition/notifyWarningMessage and did a mass replacement to remove the set****Condition calls and put whatever code was passed to that call as the code passed to notify****Message.  So there may have been a few that got a few lines of formatting messed up.

Reply:
After I did the grep, I made sure to search the whole code base for mentions of those in case I missed a few.

Reply:
There's a few places where we set the condition but never emit a message, so I wasn't sure what to do with those, so I left them be.